### DISCUSSION TITLE
Editing mode.sh

### DISCUSSION THREAD

**Discussion by TravisShepherd** [Mahmud0808/Iconify#531] [Q&A]
Trying to get iconify to work with apatch, but I'm not exactly sure how to edit the 2 options in the how-to section. How do I find the mode.sh, and how do I edit it?

**Comment by Mahmud0808** [comment]
mode.sh file is inside the overlayfs module zip, you can open and edit it with [MT Manager](https://t.me/mtmanager).

**Reply by TravisShepherd** [reply]
After editing the mode.sh line, I try to flash and get-
! Setup ext4 image failed, abort
-Error: Failed to install module script

**Reply by Mahmud0808** [reply]
Ask in overlayfs repo or support group, can't help because I am not the developer of that module. Or just skip this step and do the rest, check if everything works.
